crates/ratchet-nn/src/lib.rs

```diff
@@ -1,14 +1,16 @@
 mod embedding;
+mod groupnorm;
 mod kv_cache;
 mod linear;
 mod norm;
-mod rope;
+// mod rope;
```

Comment:
I think we need this!

Reply:
Yes sorry removed it before pulling from master 😅